renovate.json5

```diff
@@ -0,0 +1,79 @@
+// json5: https://json5.org/
+// json5 VS Code extension: https://marketplace.visualstudio.com/items?itemName=mrmlnc.vscode-json5
+// renovate configuration: https://docs.renovatebot.com/configuration-options/
+{
+    $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+
+    extends: [
+        'config:base',
+        ':automergeDisabled',
+        ':combinePatchMinorReleases',
+        ':enableVulnerabilityAlerts',
+        ':enableVulnerabilityAlertsWithLabel(security)',
+        ':label(renovate)',
+        ':rebaseStalePrs',
+        ':semanticCommits',
+        ':semanticPrefixChore',
+        ':timezone(America/Los_Angeles)',
+        ':updateNotScheduled',
+        'workarounds:typesNodeVersioning'
+    ],
+
+    enabledManagers: ['npm'],
+    branchPrefix: 'chore/renovate/',
+    platformCommit: true,
+
+    major: {
+        dependencyDashboardApproval: true
+    },
+    dependencyDashboardAutoclose: true,
+    dependencyDashboardLabels: ['renovate-dashboard'],
+    dependencyDashboardTitle: 'Renovate Dependency Dashboard',
+
+    assigneesFromCodeOwners: true,
+
+    ignorePaths: ["**/templates/**"],
+
+    packageRules: [
+        {
+            matchPackagePrefixes: ['react', '@types/react'],
+            groupName: 'dependencies [react]'
```

Comment:
we probably want to restrict packages like react using `allowedversions` as we only support V16 due to enzyme usage currently.
https://docs.renovatebot.com/configuration-options/#allowedversions

Reply:
Existing rules https://github.com/SAP/open-ux-tools/blob/main/.github/dependabot.yml#L28-L34

Reply:
Added allowedVersions: '<17.0.0' rule. Thanks.